rqt_py_common/src/rqt_py_common/message_helpers.py

```diff
@@ -227,7 +227,7 @@ def _get_rosidl_class_helper(message_type, mode, logger=None):  # noqa: C901
         return None
 
     message_info = message_type.split('/')
-    if len(message_info) != 2:
+    if len(message_info) not in [2, 3]:
```

Comment:
In which case does the `message_info` have three components?

Reply:
For services and actions, we add subfields to separate lines because there isn't currently a way of introspecting a service or action class like there is a message class. So in rqt_action for example, the first line will point to:

```
test_msgs/Fibonacci/Goal
```
Which is a rosidl msg.

Reply:
These type of names are strictly defined in ROS (1): e.g. http://wiki.ros.org/Services

In this case `<pkgname>/<msgname>` or `<pkgname>/<srvname>`. There exists no such definition for "subfields" so I don't think it is a good idea to just come up with this pattern in this package.

Either this needs to be officially defined or (what I think would make more sense) add the missing introspection information to not require this in the first place.

Reply:
I removed this change